lamindb/models/query_set.py

```diff
@@ -180,13 +180,15 @@ def get(
         return super(QuerySet, qs).get(id=idlike)  # type: ignore
     elif isinstance(idlike, str):
         qs = qs.filter(uid__startswith=idlike)
+        DOESNOTEXIST_MSG = f"no record found with uid '{idlike}'. Did you mean {registry.__name__}.get(name='{idlike}')?"
```

Comment:
We and others always capitalize error messages (as opposed to logging messages). So it should start with "No record..." ☺️

Reply:
The problem with the hint part is that it'll be confusing for transform, artifact and collection, which all don't have `name` but `key` instead; `Gene` has `symbol` instead.

You can check for whether the registry `hasattr(registry, "_name_field")` and then use this to write a hint.

I would also say "Did you maybe forget to add a keyword like ..." because of course the alternative is that the idlike argument is simply invalid because it has a typo or was deleted or the user is connected to the wrong instance (which happens a bunch).

So, we should make the message a little bit weaker so that users don't get the impression that adding a keyword is the only way out.